src/metax_api/tests/api/rest/base/views/directories/read.py

```diff
@@ -1154,6 +1154,10 @@ def test_browsing_directory_with_directory_and_file_name_and_cr_identifier_and_n
         self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
         self.assertEqual(len(response.data['directories']), 0)
 
+        response = self.client.get('/rest/directories/17/files?directory_name=2&not_cr_identifier=13')
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+        self.assertEqual(len(response.data['directories']), 0)
+
```

Comment:
is this test relevant for this PR or was it just forgotten from previous ones?

Reply:
Found bug with this case, so decided to make test for it